File: Baekjoon/silver/4378.py
import sys

input = lambda: sys.stdin.readline().rstrip()

keyboards = "`1234567890-=QWERTYUIOP[]\\ASDFGHJKL;'ZXCVBNM,./"

cor_table = {" ": " "}
for i in range(1, len(keyboards)):
    cor_table[keyboards[i]] = keyboards[i - 1]

while True:
    i = input()
    if i == "":
        break
    for char in i:
        print(cor_table[char], end="")
    print()

File: Baekjoon/silver/4378.cpp
#include <iostream>
#include <map>
using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);
    string s;
    while (getline(cin, s)) {
        string wrong = "`1234567890-=QWERTYUIOP[]\\ASDFGHJKL;\'ZXCVBNM,./";
        map<char, char> cor_table;

        cor_table.insert({' ', ' '});
        for (int i = 1 ; i < wrong.length() ; i++) {
            cor_table.insert({wrong[i], wrong[i-1]});
        }

        for (int i = 0 ; i < s.length() ; i++) {
            char letter = s[i];
            cout << cor_table[letter];
        }
        cout << '\n';
    }
}